Source/Core/DolphinQt/MemoryEngine/MemoryEngine.cpp

```diff
@@ -0,0 +1,271 @@
+#include "MemoryEngine.h"
+
+#include <QHBoxLayout>
+#include <QMenuBar>
+#include <QMessageBox>
+#include <QShortcut>
+#include <QString>
+#include <QVBoxLayout>
+#include <string>
+
+#include "../../MemoryEngine/Common/MemoryCommon.h"
+#include "../../MemoryEngine/MemoryWatch/MemWatchEntry.h"
+#include "Settings/DlgSettings.h"
+#include "Settings/DMEConfig.h"
+
+MemoryEngine::MemoryEngine(QWidget *parent) : QDialog(parent)
+{
+  setWindowTitle(tr("Memory Engine"));
+  initialiseWidgets();
+  makeLayouts();
+  //makeMenus();
```

Comment:
If this isn't used, can we get rid of it?

Or, if this is one of the reasons why this PR is still marked as draft, do you plan on enabling it?

Reply:
Yes, this should disappear. I still have some cleanup to do, but I want to make sure that I did not make mistakes on the interaction between dme and the emulator's memory.